### ISSUE TITLE
Next.js v13 app router

### ISSUE THREAD

**Issue by jim-king-2000** [gregrickaby/nextjs-github-pages#54]
Is it possible to use github pages with app router introduced in next.js v13?

**Comment by gregrickaby** [comment]
Yes. The [official example](https://github.com/vercel/next.js/tree/canary/examples/github-pages) is using the app router.

Looks like I need to update this repo too!
